packages/apollo-server-core/src/ApolloServer.ts

```diff
@@ -262,6 +279,23 @@ export class ApolloServerBase {
         throw new Error(errors.map(error => error.message).join('\n\n'));
       }
       this.schema = schema!;
+    } else if (gateway) {
+      let engineConfig =
```

Comment:
Seems this should be a `const`.  I know you'd made the suggestion that we could have a linter to help remind developers to use the correct declaration.  I fully support enabling that as a per-user linting preference in your editor if it helps you!

Reply:
Aka "I don't support enabling that as a workspace linting preference"? 🙃